sdk/storage/storage-blob/review/storage-blob.api.ts

```diff
@@ -0,0 +1,1424 @@
+  uploadBrowserData(browserData: Blob | ArrayBuffer | ArrayBufferView, options?: UploadToBlockBlobOptions): Promise<BlobUploadCommonResponse>;
+  uploadFile(filePath: string, options?: UploadToBlockBlobOptions): Promise<BlobUploadCommonResponse>;
+  uploadStream(stream: Readable, bufferSize: number, maxBuffers: number, options?: UploadStreamToBlockBlobOptions): Promise<BlobUploadCommonResponse>;
+  withSnapshot(snapshot: string): BlockBlobClient;
+}
+
+// @public
+interface BlockBlobCommitBlockListOptions {
+  abortSignal?: AbortSignalLike;
+  accessConditions?: BlobAccessConditions;
+  // (undocumented)
+  accessTier?: Models.AccessTier;
+  blobHTTPHeaders?: Models.BlobHTTPHeaders;
+  customerProvidedKey?: Models.CpkInfo;
+  metadata?: Metadata;
+  // (undocumented)
+  tier?: BlockBlobTier | string;
+}
+
+// @public
+interface BlockBlobGetBlockListOptions {
+  abortSignal?: AbortSignalLike;
+  leaseAccessConditions?: Models.LeaseAccessConditions;
+}
+
+// @public
+interface BlockBlobStageBlockFromURLOptions {
+  abortSignal?: AbortSignalLike;
+  customerProvidedKey?: Models.CpkInfo;
+  leaseAccessConditions?: Models.LeaseAccessConditions;
+  range?: Range;
+  sourceContentCrc64?: Uint8Array;
+  sourceContentMD5?: Uint8Array;
+}
+
+// @public
+interface BlockBlobStageBlockOptions {
+  abortSignal?: AbortSignalLike;
+  customerProvidedKey?: Models.CpkInfo;
+  leaseAccessConditions?: Models.LeaseAccessConditions;
+  progress?: (progress: TransferProgressEvent) => void;
+  transactionalContentCrc64?: Uint8Array;
+  transactionalContentMD5?: Uint8Array;
+}
+
+// @public (undocumented)
+enum BlockBlobTier {
+  // (undocumented)
+  Archive = "Archive",
+  // (undocumented)
+  Cool = "Cool",
+  // (undocumented)
+  Hot = "Hot"
+}
+
+// @public
+interface BlockBlobUploadOptions {
+  abortSignal?: AbortSignalLike;
+  accessConditions?: BlobAccessConditions;
+  blobHTTPHeaders?: Models.BlobHTTPHeaders;
+  customerProvidedKey?: Models.CpkInfo;
+  metadata?: Metadata;
+  progress?: (progress: TransferProgressEvent) => void;
+  tier?: BlockBlobTier | string;
+}
+
+// @public
+class BrowserPolicyFactory implements RequestPolicyFactory {
+  // WARNING: The type "BrowserPolicy" needs to be exported by the package (e.g. added to index.ts)
+  // WARNING: The type "BrowserPolicy" needs to be exported by the package (e.g. added to index.ts)
+  create(nextPolicy: RequestPolicy, options: RequestPolicyOptions): BrowserPolicy;
+}
+
+// @public
+interface ContainerAcquireLeaseOptions {
+  abortSignal?: AbortSignalLike;
+  modifiedAccessConditions?: Models.ModifiedAccessConditions;
+}
+
+// @public
+interface ContainerBreakLeaseOptions {
+  abortSignal?: AbortSignalLike;
+  modifiedAccessConditions?: Models.ModifiedAccessConditions;
+}
+
+// @public
+interface ContainerChangeLeaseOptions {
+  abortSignal?: AbortSignalLike;
+  modifiedAccessConditions?: Models.ModifiedAccessConditions;
+}
+
+// @public
+class ContainerClient extends StorageClient {
```

Comment:
This type can delete a blob and upload a block blob, but not download. Adding download here would be quite convenient I think.